docs/src/security/compliance-guidance.md

```diff
@@ -50,125 +50,8 @@ Customers are responsible for:
 * **Backups** - On Platform.sh Professional customers are responsible for all application and database backups.
 
 
-The following sections provide guidance on shared responsibilities to achieve PCI DSS compliance. The list below covers all the sections found in a typical SAQ D although you may not need this level of detail for your efforts. This section is currently using PCI DSS 3.2 as a reference.
+The following spreadsheet provides guidance on shared responsibilities to achieve PCI DSS compliance using PCI DSS 3.2 as a reference. This Excel spreadsheet was prepared by Platform.sh for informational purposes only. It is provided as a courtesy to facilitate customers’ consideration and review of the Platform.sh PCI Responsibility Summary, but the spreadsheet does not replace and is separate from the Platform.sh PCI AOC (available upon request). Customers may use the spreadsheet in conjunction with the Platform.sh PCI AOC solely to facilitate understanding elements of the report. The spreadsheet does not create any warranties, representations, contractual commitments, conditions or assurances from Platform.sh, its affiliates, vendors or licensors.
+ 
+By opening the below Excel spreadsheet, you accept and agree to these Terms of Use. If you do not wish to adhere to these Terms of Use, do not open, download, save, or otherwise access the Excel spreadsheet. 
```

Comment:
```suggestion
By opening the linked document you accept and agree to these Terms of Use. If you do not wish to adhere to these Terms of Use, do not open, download, save, or otherwise access the Excel spreadsheet.
```